intel_extension_for_transformers/llm/runtime/graph/README.md

```diff
@@ -52,29 +52,67 @@ cmake ..
 cmake --build . -j
 ```
 
-### 2. Convert LLM
+### 2. Run LLM with Python API
+
+You can use the python api to simplely run HF model.
+```python
+from intel_extension_for_transformers.transformers import AutoModel, WeightOnlyQuantConfig
+model_name = "EleutherAI/gpt-j-6b"     # support model id of HF or local PATH to model
+woq_config = WeightOnlyQuantConfig(compute_dtype="int8", weight_dtype="int4")
+model = AutoModel.from_pretrained(model_name, quantization_config=woq_config, use_llm_runtime=True)
+prompt = "Once upon a time, a little girl"
+output = model.generate(prompt, streamer, max_new_tokens=30)
```

Comment:
does `streamer` class need initialize？

Reply:
this usage is same as transformers model.generate api
so we assume consumers know how to use streamers